int32 stopThreshold
int32 travelDirection
---
string feedmsg